R/calcEmiTargetReference.R

```diff
@@ -1,42 +1,73 @@
 #' Calculate Emission Targets reference to be used when calculation Emission Targets
 #'
-#' Uses historical CEDS emissions from 1990-2015
+#' Uses historical CEDS emissions from 1990-2023
 #' CO2 (excl LU), CH4, N2O (so far no F-Gas historic time series)
-# Note: CEDS2024 does not include 'Emi|N2O|Land Use|*' variables and cannot be used.
-#' @author Aman Malik, Christoph Bertram, Oliver Richters, Rahel Mandaroux, Falk Benke
+#' @author Rahel Mandaroux, Falk Benke
 #' @seealso [calcEmiTarget()], [convertUNFCCC_NDC()]
+
 calcEmiTargetReference <- function() {
-  ceds <- calcOutput("Emissions", datasource = "CEDS2REMIND", years = 1990:2015, aggregate = FALSE)
+  # TODO Rahel: adjust documentation
```

Comment:
@RahelMA Could you adjust the description in line 3 according to the new calculation?